lib/src/specs/library.dart

```diff
@@ -30,6 +30,9 @@ abstract class Library
   /// Line comments to place at the start of the library.
   BuiltList<String> get comments;
 
+  /// A comment indicating the tool this library was generated by.
```

Comment:
Can we consider adding even more convention, and asking only for the tool name and then fill in the "Generated by" part ourselves?

Either way, I think we should have an example in the doc comment so folks know how much of the string they need to provide (currently, they need to provide the "Generated by" words in addition to the tool name, but they don't provide the `//`)

Reply:
I updated the dartdoc comments for clarity about what content to put in the message.